<launch>
    <arg name="camera_info_file" default="/home/bokorn/data/surgical/iphone_camera_info.yaml" />
    <arg name="image_folder" default="/home/bokorn/data/surgical/images/surgical_tools/" />
    <arg name="image_ext" default="png" />
    <!--arg name="frame_id" default="/camera" /-->
    <arg name="publish_rate" default="10" />

	<arg name="nof_markers" default="18" />	
	<arg name="marker_size" default="2.0" />	
    <arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
        
	<arg name="output_frame" default="/map" />

	<node name="image_publisher" pkg="pose_labeler" type="image_publisher.py" ns="image_folder" output="log">
        <param name="camera_info_file" value="$(arg camera_info_file)" />
        <param name="image_folder" value="$(arg image_folder)" />
        <param name="image_ext" value="$(arg image_ext)" />
        <param name="frame_id" value="$(arg output_frame)" />
        <!--param name="frame_id" value="$(arg frame_id)" /-->
        <param name="publish_rate" value="$(arg publish_rate)" />
        <remap from="out_image" to="/image_folder/image" />
        <remap from="out_camera_info" to="/image_folder/camera_info" />
    </node>
	
    <node name="ar_track_alvar" pkg="ar_track_alvar" type="trainMarkerBundle" respawn="false" output="screen" args="$(arg nof_markers) $(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) /image_folder/image /image_folder/camera_info $(arg output_frame)" />

</launch>
